filesystem_test.go

```diff
@@ -195,16 +195,21 @@ func (s *FilesystemSuite) TestReadFs(c *C) {
 
 func (s *FilesystemSuite) testOpenAndRead(c *C, f *Fixture, fs billy.Filesystem) {
 	for _, path := range f.contents {
-		f, err := fs.Open(path)
+		s, err := fs.Stat(path)
 		c.Assert(err, IsNil)
-		c.Assert(f, NotNil)
 
-		read, err := ioutil.ReadAll(f)
-		c.Assert(err, IsNil)
-		c.Assert(len(read) > 0, Equals, true)
+		if !s.IsDir() {
```

Comment:
Maybe there should be two different tests here.

Reply:
Yep

Reply:
I'll add new tests for directories.

Reply:
Added specific tests for nested dirs.